src/mono/wasm/build/WasmApp.LocalBuild.props

```diff
@@ -0,0 +1,51 @@
+<!--
+    WasmApp.LocalBuild* files are meant for projects that want to use
+    a local build of `dotnet/runtime` repo.
+
+    Required properties to be set:
+
+    $(RuntimeSrcDir)       - path to the runtime checkout. This should have already been
```

Comment:
Isn't having `$(WasmBuildSUpportDir)` set instead an option?

Reply:
Oops, looks like I forgot to update the comments for `WasmBuildSupportDir`. 👍